docker-compose.yml

```diff
@@ -60,13 +60,9 @@ services:
           cpus: '1'
           memory: 256M
 
+  # Should only be used for non-production deployments, see grpc-proxy/README.md for more info
```

Comment:
while you're in here fixing comments, can you prune or remove the comment at the top of the file?